staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width_test.go

```diff
@@ -410,6 +410,20 @@ func TestWorkEstimator(t *testing.T) {
 			finalSeatsExpected:        3,
 			additionalLatencyExpected: 5 * time.Millisecond,
 		},
+		{
+			name:       "creating token for service account",
```

Comment:
Can you also add a test for the same resource but without `token` subresource, to ensure that it won't be exempted?